### ISSUE TITLE
Ingress does not work

### ISSUE THREAD

**Issue by kodeine** [apache/pinot#8373]
related to PR #7997

@xiangfu0  @dianaarnos based on the template of ingress and documentation it wont work.

in documentation is below 
```
  ingress:
    v1beta1:
      enabled: false
      annotations: {}
      tls: {}
      path: /
      hosts: []
    v1:
      enabled: false
```

template file is using syntax like
```
  ingress:
    v1beta1:
      enabled: false
    v1:
      enabled: false
    annotations: {}
    tls: {}
    path: /
    hosts: []
```

**Comment by dianaarnos** [comment]
Hi, @kodeine can you give more details on how you are running things?

The helm chart version that has the ingress resource has not been released yet, so a simple `helm install apache/pinot` won't work.

**Comment by kodeine** [comment]
@dianaarnos so i am using helm, my Chart.yaml looks like

```
---
apiVersion: v2
name: apache-pinot
description: Apache Pinot
version: 0.1.0
dependencies:
  - name: pinot
    repository: https://raw.githubusercontent.com/apache/pinot/master/kubernetes/helm
    version: v0.2.4
...
```

**Comment by kodeine** [comment]
This works with new chart version just released, 0.2.5

**Comment by dianaarnos** [comment]
What do you mean by doc is wrong indentation? Can you point me to the doc so I can fix it, please?
